site/src/components/accordion.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Accordion
+description: The accordion component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - accordion
+---
+
+# Accordion
+
+Accordion is a component for grouping sections of related content that can be opened or closed. Accordion is built on top of the `<details>` HTML element.
+
+## Usage
+
+Here is an example of a standard `.accordion` usage. By default, accordion items are closed and allow multiple items to be open at once.
+
+```html
+<details class="accordion">
+	<summary>
+		<div class="accordion-header">Accordion header 1</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 1</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>
+		<div class="accordion-header">Accordion header 2</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 2</p>
+	</div>
+</details>
+<details class="accordion">
+	<summary>
+		<div class="accordion-header">Accordion header 3</div>
+	</summary>
+	<div class="accordion-content">
+		<p>Accordion panel 3</p>
+	</div>
+</details>
+```
+
+## Attributes
+
+### Open
+
+There is an option to have the accordion component load in the expanded state by adding the `open` attribute to one of the items.
```

Comment:
Nit, slight rewording, please feel free to disregard:

```suggestion
To load the accordion component in the expanded state by default, add the `open` attribute to the appropriate `<details>` element.
```

Reply:
✅Updated since I was already updating other copy.